pubsublite/create_topic.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pslite
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/pubsublite"
+)
+
+// [START pubsublite_create_topic]
+
+func createTopic(w io.Writer, projectID, region, zone, topicID string) error {
+	// projectID := "my-project-id"
+	// region := "us-central1"
```

Comment:
Could you please add a link to https://cloud.google.com/pubsub/lite/docs/locations here.